test/helpers/DeckBuilder.js

```
@@ -278,15 +282,18 @@ class DeckBuilder {
         return inPlayCards;
     }
 
-    getCapturedUnitsFromArena(arenaList) {
+    getCapturedUnitsFromArena(arenaList, ownerFilter = () => true) {
         if (!arenaList) {
             return [];
         }
         let capturedUnits = [];
         for (const card of arenaList) {
             if (typeof card !== 'string' && card.capturedUnits) {
                 for (const capturedUnit of card.capturedUnits) {
-                    capturedUnits.push(capturedUnit);
+                    let capturedUnitName = (typeof capturedUnit === 'string') ? capturedUnit : capturedUnit.card;
```

Can we go ahead and add a check here that the only permissible properties if it's an object are `card` and `owner`. This is mostly useful for catching typos (like if someone typed `onwer` it would be silently ignored)

---

Added this test

---

Example output:     TestSetupError: Invalid property in capturedUnit: card, onwer